src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,32 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
-    # TO-DO
-    
-    return merged_arr
+def merge(arrA, arrB):
+
+    left_index, right_index = 0, 0
+    result = []
+    while left_index < len(arrA) and right_index < len(arrB):
+        if arrA[left_index] < arrB[right_index]:
+            result.append(arrA[left_index])
+            left_index += 1
+        else:
```

Comment:
hehehe, implicitly, if the left_index is not smaller than the right index, than the right index is bigger! wait, what if they tie?